metis-normalization/src/main/java/eu/europeana/normalization/dates/edtf/EdtfValidator.java

```diff
@@ -90,6 +92,7 @@ private static boolean validateSpecificIntervalDates(EdtfDatePart startDatePart,
   }
 
   private static Integer adjustYearWithPrecision(Integer year, YearPrecision yearPrecision) {
+
     // TODO: 25/07/2022 Is this precision adjustment correct? How should the rounding be for positive and negatives?
```

Comment:
This is accurate and expected.
We are discarding the right most digits and we don't perform flooring.
Write a good javadoc explanation for this method.